src/components/Transaction.vue

```diff
@@ -394,7 +394,13 @@ export default Vue.extend({
         try {
           getCPKTx(this.$accessor.account.address!); /* will throw an error if no cpk tx found */
         } catch (error) {
-          this.$accessor.openModal("SignPubkey");
+          const accountID = await walletData.get().syncWallet!.getAccountId();
```

Comment:
Same way use strict type 
```ts
Const AccountId: number | undefined ...
```

Reply:
The type is automatically deduced here as well

Reply:
Ok, since typeof will cover possible changes of the external lib — here it can be kept